### taeyoonahn

**1. 연금보험료 측면 기존 유니콘 기업 데이터 분석**   
**&nbsp;&nbsp;1.1. 유니콘으로 인정받은 시점의 연금보험료 최소값     
&nbsp;&nbsp;1.2. 기업별 기간내 연금보험료 평균 상승율    
&nbsp;&nbsp;1.3. 유니콘으로 인정받은 시점의 매출/연금보험료 최소값    
&nbsp;&nbsp;1.4. 기업별 기간내 매출/연금보험료 상승율**

**2. 최종 산출물**   
**&nbsp;&nbsp;2.1. 유니콘으로 인정받은 시점의 연금보험료 최소값     
&nbsp;&nbsp;2.2. 기업별 기간내 연금보험료 평균 상승율   
&nbsp;&nbsp;2.3. 유니콘으로 인정받은 시점의 매출/연금보험료 최소값        
&nbsp;&nbsp;2.4. 기업별 기간내 매출/연금보험료 상승율**

**-연금보험료 : 연도별 월 평균 연금보험료를 뜻함**

In [149]:
#package 불러오기
import pandas as pd
import numpy as np
import seaborn as sns
%matplotlib inline
import matplotlib
import matplotlib.pyplot as plt
matplotlib.rc("font", family="Malgun Gothic")

from datetime import datetime
import statsmodels.api as sm
from statsmodels.graphics.tsaplots import plot_acf, plot_pacf
from statsmodels.tsa.arima_model import ARIMA
import warnings
warnings.filterwarnings(action='ignore')

In [150]:
#실수는 소수점 두자리까지 표기
pd.options.display.float_format = '{:.2f}'.format

In [151]:
#데이터 불러오기
df_ori=pd.read_csv('unicorn.csv',engine = "python", encoding = "CP949")
df_ori

회사ID       연매출액    년도   월   월별_연금보험료  월별_직원수
0     지피클럽   22474017  2015  11  733104.00       8
1     지피클럽   22474017  2015  12  745068.00       7
2     지피클럽   48329929  2016   1  551118.00       7
3     지피클럽   48329929  2016   2  488133.00       7
4     지피클럽   48329929  2016   3 1073632.00       7
..     ...        ...   ...  ..        ...     ...
545  옐로모바일  250700000  2019   8 1981617.00      11
546  옐로모바일  250700000  2019   9 1805970.00      11
547  옐로모바일  250700000  2019  10 1878560.00      11
548  옐로모바일  250700000  2019  11 2212953.00      12
549  옐로모바일  250700000  2019  12 1750320.00      12

[550 rows x 6 columns]

In [152]:
#데이터 정보 확인
print(df_ori.info())

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 550 entries, 0 to 549
Data columns (total 6 columns):
 #   Column    Non-Null Count  Dtype  
---  ------    --------------  -----  
 0   회사ID      550 non-null    object 
 1   연매출액      550 non-null    int64  
 2   년도        550 non-null    int64  
 3   월         550 non-null    int64  
 4   월별_연금보험료  550 non-null    float64
 5   월별_직원수    550 non-null    int64  
dtypes: float64(1), int64(4), object(1)
memory usage: 25.9+ KB
None


In [153]:
#데이터 결측치 확인
df_ori.isnull()
df_ori.isnull().sum()

회사ID        0
연매출액        0
년도          0
월           0
월별_연금보험료    0
월별_직원수      0
dtype: int64

In [154]:
#연금보험료 및 연매출액을 다루기 쉽게 데이터프레임 변경
df=df_ori[['회사ID','년도','월별_연금보험료','연매출액']]
df

회사ID    년도   월별_연금보험료       연매출액
0     지피클럽  2015  733104.00   22474017
1     지피클럽  2015  745068.00   22474017
2     지피클럽  2016  551118.00   48329929
3     지피클럽  2016  488133.00   48329929
4     지피클럽  2016 1073632.00   48329929
..     ...   ...        ...        ...
545  옐로모바일  2019 1981617.00  250700000
546  옐로모바일  2019 1805970.00  250700000
547  옐로모바일  2019 1878560.00  250700000
548  옐로모바일  2019 2212953.00  250700000
549  옐로모바일  2019 1750320.00  250700000

[550 rows x 4 columns]

In [155]:
#연금보험료를 다루기 쉽게 데이터프레임 변경 
#월별 연금보험료의 결측치를 고려해 연도별 월 평균 연금보험료로 바꿈
df_year = df.groupby(['회사ID','년도']).mean()
df_year.reset_index(inplace=True)
df_year.rename(columns = {'월별_연금보험료' : '연도별_월평균연금보험료'}, inplace = True)
df_year

회사ID    년도  연도별_월평균연금보험료        연매출액
0       무신사  2015    5921860.50    32930316
1       무신사  2016    7283040.33    47231240
2       무신사  2017    9954748.00    67721259
3       무신사  2018   16969701.42   105012384
4       무신사  2019   36105729.83   210312756
5        쏘카  2015    9692472.47    44800000
6        쏘카  2016   17036747.99    88240403
7        쏘카  2017   25767927.26   121086079
8        쏘카  2018   29599275.34   159424490
9        쏘카  2019   42750757.14   256671600
10      야놀자  2015   15675452.62    29899218
11      야놀자  2016   22816351.51    33709355
12      야놀자  2017   32040208.50    54527868
13      야놀자  2018   47430567.34    73884856
14      야놀자  2019   95933831.73   151000000
15    에이프로젠  2015    9297395.32     1823518
16    에이프로젠  2016   11437148.59    68032078
17    에이프로젠  2017   16394216.69    62178381
18    에이프로젠  2018   15201522.81    48100298
19    에이프로젠  2019   18982074.43    17857139
20  엘앤피코스메틱  2015   12470524.50   188859253
21  엘앤피코스메틱  2016   20327331.33   396118457
22  엘앤피코스메틱  2017   26900784.58   315572296
23  엘앤피코스메틱  2018   31013297.67   314078749
24  엘앤피코스메틱  2019   35541135.17   178190156
25    옐로모바일  2015    5167708.50   318200000
26    옐로모바일  2016    5553207.75   442800000
27    옐로모바일  2017   22133624.00   510500000
28    옐로모바일  2018    9245189.25   469900000
29    옐로모바일  2019    2675334.42   250700000
30   우아한형제들  2015  293398073.10    49500000
31   우아한형제들  2016  408968976.12    84850263
32   우아한형제들  2017  685929572.79   162565237
33   우아한형제들  2018  950034495.43   311535460
34   우아한형제들  2019 1473860019.92   561133434
35     지피클럽  2015     739086.00    22474017
36     지피클럽  2016    1105793.00    48329929
37     지피클럽  2017    1283563.00    49755792
38     지피클럽  2018    3796700.42   513728710
39     지피클럽  2019   11736785.00   448642100
40       쿠팡  2015  731149356.50  1133752000
41       쿠팡  2016  683545836.58  1915941000
42       쿠팡  2017  714928825.00  2681390000
43       쿠팡  2018  796258175.58  4347684000
44       쿠팡  2019 1089015207.33  7140720000
45     크래프톤  2015   35047235.00    46400000
46     크래프톤  2016   38146698.17    26519891
47     크래프톤  2017   48221011.42   282680403
48     크래프톤  2018   80043122.83  1069469987
49     크래프톤  2019  118884283.75    12102432
50       토스  2015   34006783.95       99100
51       토스  2016   58426373.25     3439965
52       토스  2017  137983534.52    20591083
53       토스  2018  247811184.05    54820876
54       토스  2019  455718589.39   118730969

**1. 연금보험료 측면 기존 유니콘 기업 데이터 분석**

**1.1. 기업별 유니콘으로 인정받은 시점의 연금보험료 최소값**

> **기업별 유니콘으로 인정받은 시점**   
>
>**-무신사 2019.11   
-쏘카 2020.10   
-야놀자 2019.6   
-에이프로젠 2019.5   
-엘앤피코스메틱 2016.1   
-옐로모바일 2014.11   
-우아한형제들 2014.12   
-지피클럽 2018.10   
-쿠팡 2014.5   
-크래프톤 2018.8   
-토스 2018.12**

In [156]:
df_unicorn=pd.DataFrame(data={'회사ID':['무신사','쏘카','야놀자','에이프로젠','엘앤피코스메틱','옐로모바일','우아한형제들','지피클럽','쿠팡','크래프톤','토스'],'유니콘_등재년도':['2019','2020','2019','2019','2016','2014','2014','2018','2014','2018','2018']})

> **기업별 유니콘으로 인정받은 시점의 연금보험료**   
>
>**-무신사 36105729.83 
-쏘카 알수없음   
-야놀자 95933831.73   
-에이프로젠 18982074.43   
-엘앤피코스메틱 20327331.33   
-옐로모바일 5167708.50(2015)   
-우아한형제들 293398073.10(2015)   
-지피클럽 3796700.42   
-쿠팡 731149356.50(2015)   
-크래프톤 80043122.83   
-토스 247811184.05**

**1.2. 기업별 기간내 연금보험료 평균 상승율**

In [157]:
#기업별 기간데이터 추출
df_period=(df_year.sort_values(['년도']).groupby(['회사ID']).년도.max())-(df_year.sort_values(['년도']).groupby(['회사ID']).년도.min())
df_period=pd.DataFrame(df_period).reset_index()
df_period.rename(columns={'년도':'기간'},inplace=True)
df_period

회사ID  기간
0       무신사   4
1        쏘카   4
2       야놀자   4
3     에이프로젠   4
4   엘앤피코스메틱   4
5     옐로모바일   4
6    우아한형제들   4
7      지피클럽   4
8        쿠팡   4
9      크래프톤   4
10       토스   4

In [158]:
#기업별 연도별_월평균보험료 변화량 추출-max년도
df_pen_max=(df_year.sort_values(['년도']).groupby(['회사ID']).년도.max())
df_pen_max=pd.DataFrame(df_pen_max).reset_index()
df_pen_max=pd.merge(df_pen_max,df_year)
df_pen_max.rename(columns = {'연도별_월평균연금보험료' : 'max년도_월평균연금보험료'}, inplace = True)
df_pen_max.rename(columns = {'년도' : '년도max'}, inplace = True)
df_pen_max.rename(columns = {'연매출액' : 'max년도_연매출액'}, inplace = True)
df_pen_max

회사ID  년도max  max년도_월평균연금보험료  max년도_연매출액
0       무신사   2019     36105729.83   210312756
1        쏘카   2019     42750757.14   256671600
2       야놀자   2019     95933831.73   151000000
3     에이프로젠   2019     18982074.43    17857139
4   엘앤피코스메틱   2019     35541135.17   178190156
5     옐로모바일   2019      2675334.42   250700000
6    우아한형제들   2019   1473860019.92   561133434
7      지피클럽   2019     11736785.00   448642100
8        쿠팡   2019   1089015207.33  7140720000
9      크래프톤   2019    118884283.75    12102432
10       토스   2019    455718589.39   118730969

In [159]:
#기업별 연도별_월평균보험료 변화량 추출-min년도
df_pen_min=(df_year.sort_values(['년도']).groupby(['회사ID']).년도.min())
df_pen_min=pd.DataFrame(df_pen_min).reset_index()
df_pen_min=pd.merge(df_pen_min,df_year)
df_pen_min.rename(columns = {'연도별_월평균연금보험료' : 'min년도_월평균연금보험료'}, inplace = True)
df_pen_min.rename(columns = {'년도' : '년도min'}, inplace = True)
df_pen_min.rename(columns = {'연매출액' : 'min년도_연매출액'}, inplace = True)
df_pen_min

회사ID  년도min  min년도_월평균연금보험료  min년도_연매출액
0       무신사   2015      5921860.50    32930316
1        쏘카   2015      9692472.47    44800000
2       야놀자   2015     15675452.62    29899218
3     에이프로젠   2015      9297395.32     1823518
4   엘앤피코스메틱   2015     12470524.50   188859253
5     옐로모바일   2015      5167708.50   318200000
6    우아한형제들   2015    293398073.10    49500000
7      지피클럽   2015       739086.00    22474017
8        쿠팡   2015    731149356.50  1133752000
9      크래프톤   2015     35047235.00    46400000
10       토스   2015     34006783.95       99100

In [160]:
#기업별 연도별_월평균보험료 변화량 추출
df_pen_diff=pd.merge(df_pen_min,df_pen_max)
df_pen_diff['변화비율']=df_pen_diff['min년도_월평균연금보험료']/df_pen_diff['max년도_월평균연금보험료']
df_pen_diff=df_pen_diff.drop(['년도min','년도max','max년도_월평균연금보험료','min년도_월평균연금보험료','min년도_연매출액','max년도_연매출액'],axis=1)
df_pen_diff

회사ID  변화비율
0       무신사  0.16
1        쏘카  0.23
2       야놀자  0.16
3     에이프로젠  0.49
4   엘앤피코스메틱  0.35
5     옐로모바일  1.93
6    우아한형제들  0.20
7      지피클럽  0.06
8        쿠팡  0.67
9      크래프톤  0.29
10       토스  0.07

In [161]:
df_pen_dif=pd.merge(df_pen_diff,df_period)
df_pen_dif['기간']=df_pen_dif['기간'].astype(float)
df_pen_dif.info()

<class 'pandas.core.frame.DataFrame'>
Int64Index: 11 entries, 0 to 10
Data columns (total 3 columns):
 #   Column  Non-Null Count  Dtype  
---  ------  --------------  -----  
 0   회사ID    11 non-null     object 
 1   변화비율    11 non-null     float64
 2   기간      11 non-null     float64
dtypes: float64(2), object(1)
memory usage: 352.0+ bytes


In [163]:
df_pen_dif['CAGR']=(df_pen_dif['변화비율'])**(1/df_pen_dif['기간'])-1
df_pen_cagr=df_pen_dif
df_pen_cagr

회사ID  변화비율   기간  CAGR
0       무신사  0.16 4.00 -0.36
1        쏘카  0.23 4.00 -0.31
2       야놀자  0.16 4.00 -0.36
3     에이프로젠  0.49 4.00 -0.16
4   엘앤피코스메틱  0.35 4.00 -0.23
5     옐로모바일  1.93 4.00  0.18
6    우아한형제들  0.20 4.00 -0.33
7      지피클럽  0.06 4.00 -0.50
8        쿠팡  0.67 4.00 -0.09
9      크래프톤  0.29 4.00 -0.26
10       토스  0.07 4.00 -0.48

**1.3. 유니콘으로 인정받은 시점의 매출/연금보험료 최소값**

In [129]:
df_spp=df_year
df_spp['연매출액/연도별_월평균연금보험료']=df_spp['연매출액']/df_spp['연도별_월평균연금보험료']
df_spp

회사ID    년도  연도별_월평균연금보험료        연매출액  연매출액/연도별_월평균연금보험료
0       무신사  2015    5921860.50    32930316               5.56
1       무신사  2016    7283040.33    47231240               6.49
2       무신사  2017    9954748.00    67721259               6.80
3       무신사  2018   16969701.42   105012384               6.19
4       무신사  2019   36105729.83   210312756               5.82
5        쏘카  2015    9692472.47    44800000               4.62
6        쏘카  2016   17036747.99    88240403               5.18
7        쏘카  2017   25767927.26   121086079               4.70
8        쏘카  2018   29599275.34   159424490               5.39
9        쏘카  2019   42750757.14   256671600               6.00
10      야놀자  2015   15675452.62    29899218               1.91
11      야놀자  2016   22816351.51    33709355               1.48
12      야놀자  2017   32040208.50    54527868               1.70
13      야놀자  2018   47430567.34    73884856               1.56
14      야놀자  2019   95933831.73   151000000               1.57
15    에이프로젠  2015    9297395.32     1823518               0.20
16    에이프로젠  2016   11437148.59    68032078               5.95
17    에이프로젠  2017   16394216.69    62178381               3.79
18    에이프로젠  2018   15201522.81    48100298               3.16
19    에이프로젠  2019   18982074.43    17857139               0.94
20  엘앤피코스메틱  2015   12470524.50   188859253              15.14
21  엘앤피코스메틱  2016   20327331.33   396118457              19.49
22  엘앤피코스메틱  2017   26900784.58   315572296              11.73
23  엘앤피코스메틱  2018   31013297.67   314078749              10.13
24  엘앤피코스메틱  2019   35541135.17   178190156               5.01
25    옐로모바일  2015    5167708.50   318200000              61.57
26    옐로모바일  2016    5553207.75   442800000              79.74
27    옐로모바일  2017   22133624.00   510500000              23.06
28    옐로모바일  2018    9245189.25   469900000              50.83
29    옐로모바일  2019    2675334.42   250700000              93.71
30   우아한형제들  2015  293398073.10    49500000               0.17
31   우아한형제들  2016  408968976.12    84850263               0.21
32   우아한형제들  2017  685929572.79   162565237               0.24
33   우아한형제들  2018  950034495.43   311535460               0.33
34   우아한형제들  2019 1473860019.92   561133434               0.38
35     지피클럽  2015     739086.00    22474017              30.41
36     지피클럽  2016    1105793.00    48329929              43.71
37     지피클럽  2017    1283563.00    49755792              38.76
38     지피클럽  2018    3796700.42   513728710             135.31
39     지피클럽  2019   11736785.00   448642100              38.23
40       쿠팡  2015  731149356.50  1133752000               1.55
41       쿠팡  2016  683545836.58  1915941000               2.80
42       쿠팡  2017  714928825.00  2681390000               3.75
43       쿠팡  2018  796258175.58  4347684000               5.46
44       쿠팡  2019 1089015207.33  7140720000               6.56
45     크래프톤  2015   35047235.00    46400000               1.32
46     크래프톤  2016   38146698.17    26519891               0.70
47     크래프톤  2017   48221011.42   282680403               5.86
48     크래프톤  2018   80043122.83  1069469987              13.36
49     크래프톤  2019  118884283.75    12102432               0.10
50       토스  2015   34006783.95       99100               0.00
51       토스  2016   58426373.25     3439965               0.06
52       토스  2017  137983534.52    20591083               0.15
53       토스  2018  247811184.05    54820876               0.22
54       토스  2019  455718589.39   118730969               0.26

> **기업별 유니콘으로 인정받은 시점**   
>
>**-무신사 2019.11   
-쏘카 2020.10   
-야놀자 2019.6   
-에이프로젠 2019.5   
-엘앤피코스메틱 2016.1   
-옐로모바일 2014.11   
-우아한형제들 2014.12   
-지피클럽 2018.10   
-쿠팡 2014.5   
-크래프톤 2018.8   
-토스 2018.12**

> **기업별 유니콘으로 인정받은 시점의 시점의 매출/연금보험료**   
>
>**-무신사 5.82   
-쏘카 알수없음   
-야놀자 1.57   
-에이프로젠 0.94   
-엘앤피코스메틱 19.49   
-옐로모바일 61.57(2015)  
-우아한형제들 0.17(2015)   
-지피클럽 135.31    
-쿠팡 1.55(2015)      
-크래프톤 13.36     
-토스 0.22**

**1.4. 기업별 기간내 매출/연금보험료 상승율**

In [141]:
#기업별 연도별_월평균보험료 변화량 추출-max년도
df_spp_max=(df_year.sort_values(['년도']).groupby(['회사ID']).년도.max())
df_spp_max=pd.DataFrame(df_spp_max).reset_index()
df_spp_max=pd.merge(df_spp_max,df_year)
df_spp_max.drop(['연도별_월평균연금보험료','연매출액'],axis=1,inplace=True)
df_spp_max.rename(columns = {'연매출액/연도별_월평균연금보험료' : 'max년도_연매출액/월평균연금보험료'}, inplace = True)
df_spp_max.rename(columns = {'년도' : '년도max'}, inplace = True)
df_spp_max

회사ID  년도max  max년도_연매출액/월평균연금보험료
0       무신사   2019                 5.82
1        쏘카   2019                 6.00
2       야놀자   2019                 1.57
3     에이프로젠   2019                 0.94
4   엘앤피코스메틱   2019                 5.01
5     옐로모바일   2019                93.71
6    우아한형제들   2019                 0.38
7      지피클럽   2019                38.23
8        쿠팡   2019                 6.56
9      크래프톤   2019                 0.10
10       토스   2019                 0.26

In [143]:
#기업별 연도별_월평균보험료 변화량 추출-min년도
df_spp_min=(df_year.sort_values(['년도']).groupby(['회사ID']).년도.min())
df_spp_min=pd.DataFrame(df_spp_min).reset_index()
df_spp_min=pd.merge(df_spp_min,df_year)
df_spp_min.drop(['연도별_월평균연금보험료','연매출액'],axis=1,inplace=True)
df_spp_min.rename(columns = {'연매출액/연도별_월평균연금보험료' : 'min년도_연매출액/월평균연금보험료'}, inplace = True)
df_spp_min.rename(columns = {'년도' : '년도min'}, inplace = True)
df_spp_min

회사ID  년도min  min년도_연매출액/월평균연금보험료
0       무신사   2015                 5.56
1        쏘카   2015                 4.62
2       야놀자   2015                 1.91
3     에이프로젠   2015                 0.20
4   엘앤피코스메틱   2015                15.14
5     옐로모바일   2015                61.57
6    우아한형제들   2015                 0.17
7      지피클럽   2015                30.41
8        쿠팡   2015                 1.55
9      크래프톤   2015                 1.32
10       토스   2015                 0.00

In [146]:
#기업별 연도별_월평균보험료 변화량 추출
df_spp_diff=pd.merge(df_spp_min,df_spp_max)
df_spp_diff['변화비율']=df_spp_diff['min년도_연매출액/월평균연금보험료']/df_spp_diff['max년도_연매출액/월평균연금보험료']
df_spp_diff=df_spp_diff.drop(['년도min','년도max','min년도_연매출액/월평균연금보험료','max년도_연매출액/월평균연금보험료'],axis=1)
df_spp_diff

회사ID  변화비율
0       무신사  0.95
1        쏘카  0.77
2       야놀자  1.21
3     에이프로젠  0.21
4   엘앤피코스메틱  3.02
5     옐로모바일  0.66
6    우아한형제들  0.44
7      지피클럽  0.80
8        쿠팡  0.24
9      크래프톤 13.01
10       토스  0.01

In [148]:
df_spp_dif=pd.merge(df_spp_diff,df_period)
df_spp_dif['기간']=df_spp_dif['기간'].astype(float)
df_spp_dif.info()

<class 'pandas.core.frame.DataFrame'>
Int64Index: 11 entries, 0 to 10
Data columns (total 3 columns):
 #   Column  Non-Null Count  Dtype  
---  ------  --------------  -----  
 0   회사ID    11 non-null     object 
 1   변화비율    11 non-null     float64
 2   기간      11 non-null     float64
dtypes: float64(2), object(1)
memory usage: 352.0+ bytes


In [166]:
df_spp_dif['CAGR']=(df_spp_dif['변화비율'])**(1/df_spp_dif['기간'])-1
df_spp_cagr=df_spp_dif
df_spp_cagr

회사ID  변화비율   기간  CAGR
0       무신사  0.95 4.00 -0.01
1        쏘카  0.77 4.00 -0.06
2       야놀자  1.21 4.00  0.05
3     에이프로젠  0.21 4.00 -0.32
4   엘앤피코스메틱  3.02 4.00  0.32
5     옐로모바일  0.66 4.00 -0.10
6    우아한형제들  0.44 4.00 -0.18
7      지피클럽  0.80 4.00 -0.06
8        쿠팡  0.24 4.00 -0.30
9      크래프톤 13.01 4.00  0.90
10       토스  0.01 4.00 -0.67

**2.최종산출물**

**2.1 유니콘으로 인정받은 시점의 연금보험료의 최소값**

> **지피클럽 3796700.42**

**2.2. 기업별 기간내 연금보험료 평균 상승율**

In [75]:
df_pen_cagr.sort_values(by='CAGR',ascending=False)

회사ID  변화량   기간  CAGR
5     옐로모바일 1.93 4.00  0.18
8        쿠팡 0.67 4.00 -0.09
3     에이프로젠 0.49 4.00 -0.16
4   엘앤피코스메틱 0.35 4.00 -0.23
9      크래프톤 0.29 4.00 -0.26
1        쏘카 0.23 4.00 -0.31
6    우아한형제들 0.20 4.00 -0.33
0       무신사 0.16 4.00 -0.36
2       야놀자 0.16 4.00 -0.36
10       토스 0.07 4.00 -0.48
7      지피클럽 0.06 4.00 -0.50

> **지피클럽 -0.50**

**2.3. 유니콘으로 인정받은 시점의 매출/연금보험료 최소값**

> **우아한 형제들 0.17**

**2.4. 기업별 기간내 매출/연금보험료 상승율**

In [167]:
df_spp_cagr.sort_values(by='CAGR',ascending=False)

회사ID  변화비율   기간  CAGR
9      크래프톤 13.01 4.00  0.90
4   엘앤피코스메틱  3.02 4.00  0.32
2       야놀자  1.21 4.00  0.05
0       무신사  0.95 4.00 -0.01
7      지피클럽  0.80 4.00 -0.06
1        쏘카  0.77 4.00 -0.06
5     옐로모바일  0.66 4.00 -0.10
6    우아한형제들  0.44 4.00 -0.18
8        쿠팡  0.24 4.00 -0.30
3     에이프로젠  0.21 4.00 -0.32
10       토스  0.01 4.00 -0.67

> **토스 -0.67**